src/libs/antares/study/scenario-builder/rules.cpp

```diff
@@ -64,7 +64,8 @@ void Rules::saveToINIFile(Yuni::IO::File::Stream& file) const
             linksNTC[i].saveToINIFile(study_, file);
         }
         // hydro levels
-        hydroLevels.saveToINIFile(study_, file);
+        hydroInitialLevels.saveToINIFileHydroLevel(study_, file, "hl,");
```

Comment:
Good job for factorization. Could you give the prefix to the constructor of hydroLevelsData ?

Reply:
done [85e2ae2](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1370/commits/85e2ae29b7d6c0b4caa1eb2bc796c36e85de066d)